main_test.go

```diff
@@ -225,6 +225,17 @@ func TestTodaySummary(t *testing.T) {
 	assert.Eventually(t, func() bool { return numCalls == 1 }, time.Second, 50*time.Millisecond)
 }
 
+func TestOfflineCount(t *testing.T) {
```

Comment:
Here it would be good to use a temporary file as offline queue like in `testSendHeartbeats` (https://github.com/wakatime/wakatime-cli/blob/develop/main_test.go#L109). Otherwise, when running it locally, it will use the default file, which might have heartbeats inside.

Reply:
Also testing it against an offline queue with heartbeats inside, would be a little more meaningful.